bin/validators.js

```diff
@@ -48,6 +48,26 @@ module.exports = {
     return Boolean(firstOption && firstOption.allowIndentationTabs);
   },
 
+  "unicorn/template-indent"({ options }) {
+    if (options.length === 0) {
+      return false;
+    }
+
+    const firstOption = options[0];
+
+    const tags = firstOption.tags || [];
```

Comment:
```suggestion
    const tags = firstOption?.tags || [];
```

Reply:
Hey,

Happy to make the change but you first said the check for its truthyness was a duplicate of the length check of the options array, implying an existing element can only be truthy if it exists

I’m not familiar with what option elements can be but I’d prefer to do the check once in the beginning instead

Reply:
Don’t worry about these code style details. Let’s see if we can cover all the tags and stuff that Prettier supports, and I can help with code style stuff before merging.